### PR TITLE
Make stepper improvements

### PR DESCRIPTION
**Pull request by JoeyDjr**
This PR improve the `Stepper` component by :  

- [x] changing the icons of the buttons of the stepper.
- [x] reducing the icon size
- [x] introducing a container for the content and the buttons and setting a background color
- [x] adapting the storybook doc

### CODE REVIEW

**Review by fredericvilcot — COMMENTED** [review]
Looks good but I found this issue (on Chrome/ macOs): 
image

File: src/ui/molecules/stepper/stepper.scss [hunk 1]
```diff
@@ -66,24 +66,38 @@ $colors: (
...
+  .okp4-stepper-container {
     width: 100%;
-    padding: 2px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    @include with-theme() {
+      background-color: themed(primary-background);
```
Comment (lolottetheclash) [inline src/ui/molecules/stepper/stepper.scss:77]: Just a question on the background: wasn't it supposed to be the same color as the cards in the faucet?

Comment (lolottetheclash) [reply src/ui/molecules/stepper/stepper.scss:77]: And I have this problem on the `Label` story: 
Capture d’écran 2022-08-23 à 08 16 31

**Review by ErikssonJoakim — APPROVED** [review]
👍

**Review by lolottetheclash — APPROVED** [review]
LGTM 👍

**Review by fredericvilcot — APPROVED** [review]

**Pull request closed without merging**